tokens/src/global/display.json

```diff
@@ -1,21 +1,25 @@
 {
-  "display": {
-    "size": {
+  "size": {
```

Comment:
I think all of the tokens here should be in `typography` category because they are related to font styles. For example `$display1-size` represents font size and `$display1-weight` - font weight (it even inherits `{typography.font.weight.bold}` value)

Reply:
Yes, you are right